pages/03-plural-features/02-k8s-upgrade-assistant/01-upgrade-insights.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Upgrade Insights
+description: How to configure and check Upgrade Insights
+---
+
+Plural integrates insights from specific cloud provider clusters. These cluster insights offer recommendations to 
+help you adhere to best practices for Kubernetes and cloud providers. They also provide information on issues that 
+could prevent upgrades and suggest ways to resolve them.
+
+## Supported providers
+Currently, the following provider is supported:
+
+- Amazon Elastic Kubernetes Service (EKS)
+
+## Configuration
+Upgrade Insights can be configured using our {% doclink to="overview_agent_api_reference" %}Custom Resource Definition (CRD){% /doclink %}. This configuration allows you to provide
+custom credentials, which are used to access the cloud provider's API and gather relevant information. For the EKS
+provider, we automatically configure node-level permissions during the cluster bootstrapping process with Plural,
+enabling our operator to retrieve upgrade insights.
+
+### Explicit credentials
+To configure Upgrade Insights with explicit credentials, you can use the following YAML configuration:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: upgrade-insights
+---
+apiVersion: v1
```

Comment:
let's not actually tell people to use this and instead suggest they set the node iam policy to have these (ignoring the secret management annoyance).  I did it in the bootstrap terraform already.

Reply:
(ah you did that below, let's just reorder them then)